diff --git a/drivers/net/wireless/realtek/rtl88x2bu/Makefile b/drivers/net/wireless/realtek/rtl88x2bu/Makefile
index dacda03..cd1cae2 100644
--- a/drivers/net/wireless/realtek/rtl88x2bu/Makefile
+++ b/drivers/net/wireless/realtek/rtl88x2bu/Makefile
@@ -102,7 +102,7 @@ CONFIG_RTW_UP_MAPPING_RULE = tos
 CONFIG_RTW_MBO = n
 ########################## Android ###########################
 # CONFIG_RTW_ANDROID - 0: no Android, 4/5/6/7/8/9/10/11 : Android version
-CONFIG_RTW_ANDROID = 0
+CONFIG_RTW_ANDROID = 11
 
 ifeq ($(shell test $(CONFIG_RTW_ANDROID) -gt 0; echo $$?), 0)
 EXTRA_CFLAGS += -DCONFIG_RTW_ANDROID=$(CONFIG_RTW_ANDROID)
